parachains/common/src/xcm_config.rs

```diff
@@ -57,9 +57,61 @@ impl<Location: Get<MultiLocation>> ContainsPair<MultiAsset, MultiLocation>
 	for ConcreteNativeAssetFrom<Location>
 {
 	fn contains(asset: &MultiAsset, origin: &MultiLocation) -> bool {
-		log::trace!(target: "xcm::filter_asset_location",
+		log::trace!(target: "xcm::contains",
 			"ConcreteNativeAsset asset: {:?}, origin: {:?}, location: {:?}",
 			asset, origin, Location::get());
 		matches!(asset.id, Concrete(ref id) if id == origin && origin == &Location::get())
 	}
 }
+
+/// Accepts an asset if it is a native asset from a System Parachain.
+pub struct NativeAssetFromSiblingSystemParachain;
+impl ContainsPair<MultiAsset, MultiLocation> for NativeAssetFromSiblingSystemParachain {
+	fn contains(asset: &MultiAsset, origin: &MultiLocation) -> bool {
+		log::trace!(target: "xcm::contains", "NativeAssetFromSiblingSystemParachain asset: {:?}, origin: {:?}", asset, origin);
+		let is_system_para = match origin {
```

Comment:
@NachoPal @joepetrowski 
I talked about something similar with Joe during decoded, 
I have started to work on some cleaning and refactoring in polkadot repo for `is_system < 2000` (hope today will have some draft)

Reply:
Ok, just bare in mind that I think we want it to be merged for the next release.

Reply:
we can use here `if ParaId::from(id).is_system()`
but it will then depend on my polkadot PR :)

Reply:
ok, PR is ready, but there are few questions:
https://github.com/paritytech/polkadot/pull/7488

Reply:
@NachoPal
ok, I would go with this `id < 2000` maybe with some TODO that soon it will be changed to `if ParaId::from(id).is_system()`, I dont want to block this with my PR, which will takes more time

and just nit to rename:
**NativeAssetFrom**SiblingSystemParachain
to
**ConcreteNativeAssetFrom**SiblingSystemParachain  
(to be compatible with  ConcreteNativeAssetFrom above)